_aggregations/metric/cardinality.md

```diff
@@ -9,9 +9,25 @@ redirect_from:
 
 # Cardinality aggregations
 
-The `cardinality` metric is a single-value metric aggregation that counts the number of unique or distinct values of a field.
+The `cardinality` metric is a single-value metric that counts the number of unique or distinct values of a field.
 
-The following example finds the number of unique products in an eCommerce store:
+
+Cardinality count is approximate. See [Controlling precision](#controlling-precision).
+
+## Parameters
+
+The `cardinality` aggregation takes the following parameters.
+
+| Parameter             | Required/Optional | Data type       | Description |
+| :--                   | :--               |  :--            | :--         |
+| `field`               | Required          | String          | The field for which the cardinality is estimated. |
+| `precision_threshold` | Optional          | Numeric         | Count below which good accuracy is expected. See [Controlling precision](#controlling-precision).     |
+| `execution_hint`      | Optional          | String          | How to run the aggregation. Valid values are `ordinals` and `direct`. |
+| `missing`             | Optional          | Same as `field`'s type | Bucket to assign missing instances of the field. If not given, missing values are ignored. |
+
+## Example
+
+The following example finds the number of unique product IDs in the commerce sample data:
 
```

Comment:
Line 30: Does "the sample e-commerce data" refer to "the OpenSearch Dashboards sample e-commerce data"?

Reply:
Yes.

Reply:
@dwelsch-esi Let's go ahead and add that for clarity.